EazyShop/LoginPage.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  LoginPage.swift
+//  EazyShop
+//
+//  Created by Cesar Rojas on 7/31/24.
+//
+
+import SwiftUI
+
+struct LoginPage: View {
+    @State private var email: String = ""
+    @State private var password: String = ""
+    @State private var nextView: Bool = false
+    @State private var isValidEmail: Bool = true
+    
+    var body: some View {
+        VStack(spacing: 20) {
+            // Title
+            Text("Login")
+                .font(.custom("Metropolis-ExtraBold", size: 34)).bold()
+                .bold()
+                .frame(maxWidth: .infinity, alignment: .leading)
+                .padding(.bottom, 53)
+            
+            VStack {
+                VStack(alignment: .leading) {
+                    Text("Email")
+                        .font(.custom("Metropolis-Regular", size: 11))
+                        .foregroundColor(.gray)
+                        .padding(.leading, 10)
+                    HStack {
+                        // Email Field
+                        TextField("", text: $email)
+                            .onChange(of: email) { newValue in
+                                isValidEmail = newValue.isValidEmail()
+                        }
+                            .font(.custom("Metropolis-Regular", size: 14))
+                            .padding(.leading, 10)
+                    }
+                }
+                .frame(height: 64)
+                .overlay(RoundedRectangle(cornerRadius: 0).stroke(isValidEmail ? Color(.systemGray4) : Color(hex: "F01F0E"), lineWidth: 2))
+                
+                if !isValidEmail {
```

Comment:
move this to viewModel. Dont put any logic in UI. Since we are following MVVM.

Reply:
Moved to viewModel!